lib/actions/device.coffee

```diff
@@ -61,15 +61,12 @@ exports.info =
 	description: 'list a single device'
 	help: '''
 		Use this command to show information about a single device.
-
 		Examples:
-
 			$ resin device MyDevice
 	'''
 	permission: 'user'
 	action: (params, options, done) ->
-		resin.models.device.get params.name, (error, device) ->
-			return done(error) if error?
+		resin.models.device.get(params.name).then (device) ->
```

Comment:
Notice that now most of the device functions in the SDK take UUIDs instead of plain names. In those cases, can you rename the param to `uuid` here and in the capitano command definition, as well as modifying the command example above to use a fake uuid?